src/main/java/gg/solarmc/clans/events/StatsEvent.java

```diff
@@ -0,0 +1,57 @@
+package gg.solarmc.clans.events;
+
+import gg.solarmc.loader.clans.Clan;
+import gg.solarmc.loader.clans.ClansKey;
+import org.bukkit.Server;
+import org.bukkit.attribute.Attribute;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.EntityDamageByEntityEvent;
+import org.bukkit.event.entity.EntityRegainHealthEvent;
+import org.bukkit.event.entity.PlayerDeathEvent;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+public class StatsEvent implements Listener {
```

Comment:
I'm somewhat concerned about our stats tracking, I'll explain in the overall review comment.